hack/validate-licence.bash

```diff
@@ -8,5 +8,5 @@
 set -e
 
 go build ./hack/licenseok
-find . -path ./vendor -prune -o -type f -name "*.go"\
+find . -path ./vendor -prune -o -regex ".+\.pb\.go$" -prune -o -type f -regex ".*\.\(go\|proto\)$"\
```

Comment:
The intention was to include `*.proto` files and exclude the generated `*.pb.go` files.  I stumbled my way into this, but perhaps there is a simpler or more readable way to do so.

Reply:
i have no immediate ideas about better ways of doing this; i'm fine with a slightly wonky regex.